sites/example-project/src/components/viz/ErrorChart.svelte

```diff
@@ -3,62 +3,13 @@
     export let chartType;
 </script>
 
-<div width=100% class="error">
-    <div class="wrapper">
-        <div class="chart-type">{chartType}</div>
+<div class="grid [grid-template-rows:auto] grid-cols-[100%] justify-center bg-red-50 text-grey-700 font-ui font-normal rounded border border-red-100 min-h-[150px] box-content py-5 px-[30px] my-5 print:break-inside-avoid">
```

Comment:
I think there's a slight regression here. The error chart should respect the width of it's container. So for charts themselves it will be the full width of the article, but in the case of BigValue (and potentially other components in the future), it should only take up the default width of that component. Compare [this branch](https://evidence-development-workspace-mtkzjo7f2-evidence-dev.vercel.app/text-and-metrics/big-value) with the [current state](https://evidence-development-workspace-git-table-download-evidence-dev.vercel.app/text-and-metrics/big-value)

Reply:
This has been updated in new commits. Marking as resolved.